test/unit/disposable_common.dart

```diff
@@ -272,12 +270,179 @@ void testCommonDisposable(Func<StubDisposable> disposableFactory) {
     });
 
     test(
-        'should return ManagedStreamSubscription that returns null when an'
+        'should throw if stream completes with an error and there is no '
+        'onError handler', () {
+      runZoned(() {
+        // ignore: close_sinks
+        var controller = new StreamController<Null>();
+        // ignore: cancel_subscriptions
+        var subscription = disposable.listenToStream(controller.stream, (_) {});
+        controller.addError(new Exception('intentional'));
```

Comment:
🌵 
``` dart
var controller = new StreamController<Null>();
controller.addError(new Exception('intentional'));
```
Could be moved outside of the `runZoned` call. Also it seems like the `subscription` var isn't used.